script.js

```diff
@@ -0,0 +1,152 @@
+function Book(title, author, pages, isbn, status) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.isbn = isbn;
+  this.status = status;
+}
```

Comment:
- [ ] Apart from this constructor function. Kindly change every **other** regular function into es6 arrow function syntax.